starter-code/routes/posts.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+
+const User = require('../models/user');
+const Post = require('../models/post');
+
+router.get('/:userId/posts/new', (req, res, next) => {
+  const userInfo = { id: req.params.userId };
+  res.render('posts/new', userInfo);
+});
+
+router.post('/:userId/posts', (req, res, next) => {
+  const newPost = new Post({
+    title: req.body.postTitle,
+    content: req.body.content,
+    _creator: req.params.userId
+  });
+
+  newPost.save((err) => {
```

Comment:
Don't you need to check before if the person who is creating the post is the actual user?